common.blocks/button/button.css

```diff
@@ -50,3 +54,51 @@
 .button_action_download:hover {
     background: #6cae11;
 }
+
+@media screen and (max-width: 773px) {
+    .icon_ico_play {
+        width: 27px;
+        height: 27px;
+        background-size: cover;
+    }
+
+    .button_size_l {
+        font-size: 1.6em;
+        line-height: 50px;
+        padding-bottom: 6px;
+    }
+
+    .button_size_m {
+        font-size: 1.6em;
+        line-height: 30px;
+        padding-bottom: 0;
+    }
+}
+/* Portrait */
+@media only screen
+and (min-device-width: 320px)
+and (max-device-width: 773px)
+and (orientation: portrait) {
+    .page_mobile-height_small .button {
+        font-size: 14px;
```

Comment:
On devices having pixel ratio equal to 2, such as iphone, it is easier to adjust the value in px